### DISCUSSION TITLE
Web3auth integration

### DISCUSSION THREAD

**Discussion by 3scava1i3r** [wevm/wagmi#607] [Ideas]
I wish to work on web3auth integration with wagmi as a potential wallet connector
You can check their project at - https://web3auth.io/

**Comment by tmm** [comment]
Check out this [guide](https://wagmi.sh/examples/custom-connector) on how to create a custom connector.

**Reply by 3scava1i3r** [reply]
Will check it out thanks,is there any discord server we can chat on?

**Reply by tmm** [reply]
No Discord at the moment. Easier for the core team if everything is async and open right now.

**Reply by 3scava1i3r** [reply]
Saw the plugin for magic and had great similarities,just one issue do we have to manually build the packages like for react and core and then import for it to work as a connector?

**Reply by 3scava1i3r** [reply]
> No Discord at the moment. Easier for the core team if everything is async and open right now.

so i had this issue while trying to test i using the react dev example,it says module not found and which it should legit do,but how do i get it there any docs on building the files so that i easily import the same?

**Reply by billtlee** [reply]
I would love to see a web3auth connector as well.

**Comment by recidive** [comment]
@3scava1i3r any progress on that? Can we join forces, maybe?

**Comment by YZhenY** [comment]
https://www.npmjs.com/package/@web3auth/web3auth-wagmi-connector

**Reply by YZhenY** [reply]
theres one we've done!
